gradle.properties

```diff
@@ -2,9 +2,9 @@ org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 
 mod_version=1.3.2
-mc_version=1.16.2
-forge_version=33.0.37
-mappings_version=20200723-1.16.1
+mc_version=1.16.3
+forge_version=34.1.5
+mappings_version=20200514-1.16
```

Comment:
Why using older mappings as before? Latest is 20200916-1.16.2

Reply:
oops my bad. Fixed.